src/app/components/main-page/main-page.component.html

```diff
@@ -35,13 +35,13 @@ <h3 class="centered">Top rental bikes</h3>
         <span class="card__category"> ₴ 100 / day</span> <br>
         <span class="card__category"> ₴ 400 / week</span>
```

Comment:
we do not have implemented logic for discounts.

Reply:
I know it is for cool look

Reply:
@Nazarii77 do you insist on keeping it?

Reply:
yes, we presented with this one today